.github/workflows/build_canary.yml

```diff
@@ -13,61 +13,114 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build and push operator image
-        uses: docker/build-push-action@v1.1.1
+      - name: Set up tags and refs
+        id: prep
+        run: |
+          SHA="sha-${GITHUB_SHA::8}"
+          echo ::set-output name=sha::${SHA}
+          echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+
+      - name: Set up Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GHCR
+        uses: docker/login-action@v1
         with:
           # Username used to log in to a Docker registry. If not set then no login will occur
-          username: ${{ secrets.GHCR_AUTH_USER }}
+          username: ${{ github.repository_owner }}
```

Comment:
why not use the secret still?

Reply:
No particular reason, just wanted to make the script simples and have less external secrets, but I can change it

Reply:
sounds good 👍 thanks for explaining 😄